.changeset/pink-gorillas-hug.md

```diff
@@ -0,0 +1,5 @@
+---
+'@talend/react-components': patch
```

Comment:
This is an API break though, with possible breaking side-effects. You guys are sure?

Reply:
Sure, I can believe it's not necessary. But if it has been done somewhere, then this PR becomes breaking, not patch.

Reply:
I'm confident that this property is not used.
Everything already need is passed aside, spreading all the row data to the "title" button has no usage in our app afaik.

Thanks for raising this point.

Reply:
Yes, this was an error at first.
The action represents the title of the tile that is restricted. This idea is not to open it to customisation.
And here we spread all the list's item properties as props, which doesn't make sense. In practice we end up with tons of unexpected props in Action component input, that won't be used.

Reply:
I found a side-effect : it removes the `name` attribute of CellTitle ActionButton button tag leading to break selectors.